src/utils/clip_utils.py

```diff
@@ -28,6 +28,57 @@ def square_clip(xy_points, bounds):
     return inds
 
 
+@jit(nopython=True)
+def circle_clip(points, center, radius):
+    """
+    Clip all points within a circle (or unbounded cylinder).
+
+    Parameters
+    ----------
+    points : array of shape (n_points, 2)
+        The points.
+    center : tuple (x, y)
+        Center point of the circle.
+    radius : float
+        Radius of the circle.
+
+    Returns
+    -------
+    A boolean mask with True entries for all points within the circle.
+    """
+    clip_mask = (np.power((points[:, 0] - center[0]), 2)
```

Comment:
Bij square_clip gebruiken we xy_points['x'] ipv points[:, 0]. Idee is hetzelfde

Reply:
Alles nu herschreven!